setup.py

```diff
@@ -327,7 +327,7 @@ def make_release_tree(self, base_dir, files):
         "numpy>=1.21",
         "packaging>=20.0",
         "pillow>=6.2.0",
-        "pyparsing>=2.3.1",
```

Comment:
I do not think we should pin it here yet.  The "happy paths" still work, it is the error messages that are broken.